package/gluon-status-page/files/lib/gluon/status-page/www/cgi-bin/dyn/neighbours-nodeinfo

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+echo 'Access-Control-Allow-Origin: *'
+
+batctl if | grep -q "^$QUERY_STRING:"  || exit 1
```

Comment:
I'm going to replace that line with

    batctl if | cut -d: -f1 | grep -qxF "$QUERY_STRING" || exit 1

with a force push.